ModelDesignWizard/ModelDesignWizardDialog.cpp

```diff
@@ -30,6 +30,7 @@
 #include "ModelDesignWizardDialog.hpp"
 #include "Buttons.hpp"
 #include "OSQuantityEdit.hpp"
+#include "Assert.hpp"
```

Comment:
I got this to build and run, seems like this would be best integrated as a modal pop up dialog in the app rather that a standalone executable.  Is that what you are thinking?

Reply:
Yes. I started on the OSApp repo, but testing got really annoying because the build+test times were too long. Build, launch, wait for App to initialize (that's the real bottleneck), test, rinse and repeat. So I figured it was a good idea to prototype outside of the App first.